core/src/main/res/layout/activity_history.xml

```diff
@@ -7,6 +7,19 @@
 
   <include layout="@layout/layout_standard_app_bar" />
 
+  <TextView
+    android:id="@+id/no_history"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:text="@string/no_history"
+    android:textAppearance="@style/TextAppearance.KiwixTheme.Headline5"
+    android:gravity="center"
```

Comment:
gravity center is still on this textView

Reply:
@macgills apologies. I forgot to remove it. I've removed it now. Is the pr fine now?

Reply:
Yup, I think it is there, good job